aidial_adapter_openai/dalle3.py

```diff
@@ -75,19 +73,17 @@ async def generate_stream(
     yield build_chunk(id, "stop", {}, created, True, usage=IMG_USAGE)
 
 
-def get_user_prompt(data: Any):
-    if (
-        "messages" not in data
-        or len(data["messages"]) == 0
-        or "content" not in data["messages"][-1]
-        or not data["messages"][-1]
-    ):
-        raise DialException(
-            "Your request is invalid", 400, "invalid_request_error"
+def get_user_prompt(data: Any) -> str:
+    try:
+        prompt = data["messages"][-1]["content"]
+        if isinstance(prompt, str):
+            raise ValueError("Content isn't a string")
+        return prompt
+    except Exception:
+        raise RequestValidationError(
```

Comment:
You can use
```python

except Exception as e:
    raise RequestValidationError(...) from e
```

To avoid 
```
During handling of the above exception, another exception occurred:
```
in logs

Reply:
fixed